typescript/cli/src/commands/warp.ts

```diff
@@ -82,8 +87,21 @@ export const apply: CommandModuleWithWriteContext<{
       demandOption: false,
     },
     strategy: { ...strategyCommandOption, demandOption: false },
+    'receipts-dir': {
+      type: 'string',
+      description: 'The directory to output transaction receipts.',
```

Comment:
this is this a directory as opposed to a file path?

Reply:
Yeah, there could be multiple receipts so can't just supply a single filepath

Reply:
not sure I follow what you mean by multiple receipts
is this a consequence of not submitting transactions as batches?